.github/workflows/phpstan/autoload.php

```diff
@@ -16,6 +16,8 @@
 require_once _PS_ROOT_DIR_ . '/install-dev/classes/exception.php';
 require_once _PS_ROOT_DIR_ . '/install-dev/classes/session.php';
 require_once _PS_ROOT_DIR_ . '/var/SymfonyRequirements.php';
+require_once _PS_ROOT_DIR_ . '/admin-dev/functions.php';
+require_once _PS_ROOT_DIR_ . '/vendor/paragonie/random_compat/lib/byte_safe_strings.php';
```

Comment:
I don't understand this one? why, isn't it in the autoload.php?

Reply:
I try without. And that generate an alert.

Reply:
We should find why it fails, not adding this line without a valid reason :)

Reply:
Fixed